api-specificatie/openapi.yaml

```diff
@@ -123,11 +157,19 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/VolgindicatiePersonen"
+                $ref: "#/components/schemas/VolgindicatiePersoon"
```

Comment:
waarom levert een DELETE een response 200 met body terug? Wanneer gebruik je die versus de 204?

Reply:
Ik heb er even naar gezocht en diverse bronnen geven aan dat een 200 op een DELETE wel degelijk van toepassing kan zijn. Het volgende geldt dan:

> A successful response of DELETE requests SHOULD be HTTP response code 200 (OK) if the response includes an entity describing the status, 202 (Accepted) if the action has been queued, or 204 (No Content) if the action has been performed but the response does not include an entity.

De vraag is dus of we verwachten dat er in dit geval ooit een entity in de response zal worden teruggegeven.

Reply:
Klopt, maar de nu gedefinieerd 200 response beschrijft niet de status, maar levert de resource die niet meer bestaat. 

Ik denk dat het enige dat de gebruiker moet weten is dat de volgindicatie succesvol is verwijderd. Status 204 doet exact dat.

Reply:
in #19 wordt operatie DELETE verwijderd